.coveragerc

```diff
@@ -2,10 +2,15 @@
 branch = True
 source = RestrictedPython
 omit =
+    bootstrap.py
```

Comment:
This seems to be not needed here as it is outside the directories configured for the coverage.